test/conformance/walk.go

```diff
@@ -85,13 +87,16 @@ func (v *visitor) convertToConformanceData(at *ast.BasicLit) {
 	cd.Description = ""
 	for _, line := range lines {
 		line = strings.TrimSpace(line)
-		if strings.HasPrefix(line, "Testname:") {
-			line = strings.TrimSpace(line[9:])
-			cd.TestName = line
+		if sline := regexp.MustCompile("Testname\\s*:\\s*").Split(line, -1); len(sline) == 2 {
```

Comment:
For correctness and to maintain behavior with HasPrefix should we use the start of line indicator?
(Eg. "^Testname\\s*:\\s*"

Reply:
I agree, it has to be the start of the line, fixed.